apollos-church-api/config.yml

```diff
@@ -78,25 +55,13 @@ PASS:
     SIGNER_CERT: ${PASS_SIGNER_CERT}
     SIGNER_KEY: ${PASS_SIGNER_KEY}
     SIGNER_KEY_PASSPHRASE: ${PASS_SIGNER_PASSPHRASE}
-ANALYTICS:
-  # Filling out these fields will automatically enable analytics tracking
-  # on the accompanying service
-  SEGMENT_KEY:
-  GA_ID:
-  USE_ROCK: false
 BIBLE_API:
   KEY: ${BIBLE_API_KEY}
   BIBLE_ID:
     WEB: '9879dbb7cfe39e4d-01'
     KJV: 'de4e12af7f28f599-02'
 CLOUDINARY:
-  # Cloudinary is used to improve image loading
```

Comment:
I think this is another comment that could stay :) The key is pretty ambiguous if you aren't familiar with cloudinary

Reply:
I removed this comment (and others) because I describe what it is and how to get it (ENV variable) in the cloudinary set up doc

Reply:
so you never actually look at this comment anyway